src/engraving/rw/write/measurewrite.cpp

```diff
@@ -55,6 +55,9 @@ void MeasureWrite::writeMeasure(const Measure* measure, XmlWriter& xml, WriteCon
         xml.tag("multiMeasureRest", measure->m_mmRestCount);
     }
     if (writeSystemElements) {
+        if (!MScore::testMode && !measure->score()->isPaletteScore()) {
```

Comment:
Here too, I think we might want to create a more specific flag. While it needs to be disabled for current tests, because updating all those test files is not useful, it could perhaps be enabled for future tests.

Reply:
Agree. I'll come back to this when I do the aforementioned work on EIDs